source/includes/_request_and_response.md

```diff
@@ -2,7 +2,7 @@
 
 ## API Endpoint
 
-`https://office-api.evocalize.com/`
+`https://partner-api-production.evocalize.com/`
```

Comment:
We can shorten this it to `partner-api` instead of `partner-api-production` if we really want to but i'm honestly ok exposing this to partners